add recipe for ninrod/evil-string-inflection package

ninrod opened the pull request:
### Brief summary of what the package does

provides an evil operator to cycle text objects through various case formats: kebab, camel, snake, UPPER.

### Direct link to the package repository

https://github.com/ninrod/evil-string-inflection

### Your association with the package

I'm the author

### Relevant communications with the upstream package maintainer

**none needed**

### Checklist

Please confirm with `x`:

- [x ] The package is released under a [GPL-Compatible Free Software License](https://www.gnu.org/licenses/license-list.en.html#GPLCompatibleLicenses). 
- [ x] I've read [CONTRIBUTING.md](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.md)
- [ x] I've used the latest version of [package-lint](https://github.com/purcell/package-lint) to check for packaging issues, and addressed its feedback
- [x ] My elisp byte-compiles cleanly
- [ x] `M-x checkdoc` is happy with my docstrings
- [ x] I've built and installed the package using the instructions in [CONTRIBUTING.md](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.md)

tarsius commented on recipes/snakecamelfy:
```diff
@@ -0,0 +1 @@
+(exato :repo "ninrod/snakecamelfy" :fetcher github)
```
`exato` has to be replaced.

ninrod replied:
:man_facepalming:

ninrod replied:
Ok, done!

ninrod commented:
@tarsius, should be fine now.

tarsius commented:
The build error is due to an unrelated change, which is fixed now. Please rebase your commit so that the tests can be run for your change. If you use Magit you can do so using <kbd>f u</kbd> to fetch, <kbd>r u</kbd> to rebase, and <kbd>P - f p</kbd> to force push.

By the way you should also do that fixing an issue in your recipe submission. So instead of adding a second commit "fix copy/paste leftover", you should just <kbd>c e</kbd> to amend and <kbd>P - f p</kbd> to force push. @purcell that's what you would prefer, right?

ninrod commented:
@tarsius and @purcell, I've implemented the rebase and squashed the commits. but it seems that there's an issue in travis regarding the build [path](https://travis-ci.org/melpa/melpa/builds/308348271#L1036).

tarsius commented:
Turns out my fix was not complete... still working on it.

tarsius commented:
Please try again it should work now.

ninrod commented:
@tarsius, do you need anything else from me regarding this pull?

tarsius commented:
No, the tests pass, which means the things that I broke are no longer broken and also that nothing that could be detected by the tests is broken in your submission :wink:

purcell commented:
Great! Same comments here as in #5149, re. package naming and key customisation.

purcell commented:
Also, you might consider using the [string-inflection package](`https://github.com/akicho8/string-inflection/blob/e644c8bf2a5e36ddadcfe4de657588c00fd368dd/string-inflection.el) to avoid reinventing this code.

ninrod commented:
> Also, you might consider using the string-inflection package to avoid reinventing this code.

This was a game changer. I had to rewrite and rename the whole package, got more functionality than I ever hoped and in the process end up with just 4 lines of code. Talk about reuse. 👍 

thank you very much @purcell. I'd appreciate a new review.
